src/Common/PlanetParameters/PlanetaryConstants.jl

```diff
@@ -0,0 +1,83 @@
+"""
+    PlanetaryConstants
+
+Module containing physical constants and parameters characterizing the planet.
+"""
+module PlanetaryConstants
+
+using ..UniversalConstants
+
+export PlanetConstants,
+       AbstractPlanetConstants,
+       EarthConstants
+       # MarsConstants
+
+Base.@kwdef struct PlanetConstants{FT<:AbstractFloat}
+  k_Karman::FT        # Von Karman constant (1)
```

Comment:
Can add docstings
```suggestion
  "Von Karman constant (1)"
  k_Karman::FT
```

Reply:
Why not just add the default values here, i.e.
```
```suggestion
  k_Karman::FT= 0.4
```

Reply:
I'm not sure this is a good idea because it will be difficult to catch if you accidentally miss a keyword for one of the other constructors (that's why I don't prescribe any for the default).

Reply:
This way, if a constructor for a particular planet is incomplete, we'll know right away.